src/items/static-items.md

```diff
@@ -12,21 +12,27 @@ r[items.static.syntax]
 >   allowed semantically within `extern` blocks.
 
 r[items.static.intro]
-A *static item* is similar to a [constant], except that it represents a precise
-memory location in the program. All references to the static refer to the same
-memory location.
+A *static item* is similar to a [constant], except that it represents an allocated object in the
+program that is initialized with the initializer expression. All references and raw pointers to the
+static refer to the same allocated object.
 
 r[items.static.lifetime]
-Static items have the `static` lifetime, which outlives all
-other lifetimes in a Rust program. Static items do not call [`drop`] at the
-end of the program.
+Static items have the `static` lifetime, which outlives all other lifetimes in a Rust program.
+Static items do not call [`drop`] at the end of the program.
+
+r[items.static.storage-disjointness]
+If the `static` has a size of at least 1 byte, this allocated object is disjoint from all other such
+`static` objects as well as heap-allocated and stack-allocated variables. However, the storage of
+immutable `static` items can overlap with objects that do not themselves have a unique address, such
```

Comment:
“Immutable `static` item” could be interpreted as meaning a static item without the `mut` keyword, but it must also exclude interior-mutable memory. Perhaps this should be clarified (or perhaps it doesn’t matter since this is a “can”, not “must”).

Reply:
Yes we should have this as a shared definition somewhere that we can reference.